### ISSUE TITLE
View the input parameters used for a scenario run from the frontend

### ISSUE THREAD

**Issue by LukeParky** [GeospatialResearch/Digital-Twins#148]
Blocked by #147 
This is the front-end task for #147

**Comment by LukeParky** [comment]
You meant to close #147 @Martin20494

**Comment by LukeParky** [comment]
This issue is not completed. It is a separate issue

**Comment by caseyli92** [comment]
@LukeParky, I just want to confirm that, as we discussed previously, we no longer need to export the input parameters from the front-end. Instead, we just need to be able to view them from the front-end along with the associated scenario run.

**Comment by LukeParky** [comment]
Yes I think that is acceptable
